main.cpp

```diff
@@ -401,90 +445,189 @@ struct Engines
     float loadingTime = 45;
-    // 3 things it can do:
-    //     1) Transport
+
     void transport();
-    //     2) Store
-    void store();
-    //     3) Secure
-    void secure();
+    void store(int amountOfCargo);
+    void secure(bool isDoorEnclosed);
 };
 
+void CargoSpace::transport()
+{
+    std::cout << "Cargo is being transported." << std::endl;
+}
+
+void CargoSpace::store(int amountOfCargo)
+{
+    if(amountOfCargo <= capacity)
+    {
+    std::cout << amountOfCargo << " units of cargo stored." << std::endl;
+    }
+    else
+    {
+    std::cout << "Cargo exceeds maximum capacity." << std::endl;
+    }
+}
+
+void CargoSpace::secure(bool isDoorEnclosed)
+{
+    if(isDoorEnclosed)
+    {
+    std::cout << "Cargo is secured." << std::endl;
+    }
+    else
+    {
+    std::cout << "Cargo is not secured. Please close the door." << std::endl;
+    }
+}
+
+// UDT 8
 struct Tail
 {
-    // 5 properties:
-    //     1) Tail length (int)
     int length = 55;
-    //     2) Tail width (int)
     int width = 85;
-    //     3) Elevator angle (float)
-    float elevatorAngle = 45.f;
-    //     4) Stabilizer angle (float)
-    float stabilizerAngle = 45.f;
-    //     5) Yaw control width (int)
+    int elevatorAngle = 45;
+    int stabilizerAngle = 45;
     int yawControlWidth = 30;
-    // 3 things it can do:
-    //     1) Create stability
+
     void stabilize();
-    //     2) Create lift
     void createLift();
-    //     3) Control side to side movement
     void controlSideToSide();
 };
 
+void Tail::stabilize()
+{
+    if(elevatorAngle != 0 || stabilizerAngle != 0)
+    {
+    elevatorAngle = 0;
+    stabilizerAngle = 0;
+    }
+}
+void Tail::createLift()
+{
+    if(elevatorAngle < 15)
+    {
+    elevatorAngle = 15;
+    }
+}
+void Tail::controlSideToSide()
+{
+    if(yawControlWidth < 30)
+    {
+    yawControlWidth += 10;
+    }
+}
+
+// UDT 9
 struct Fuselage
 {
-    // 5 properties:
-    //     1) Length of fuselage (int)
     int length = 30;
-    //     2) Width of fuselage (int)
     int width = 15;
-    //     3) Height of fuselage (float)
     float height = 19.f;
-    //     4) Capacity of fuselage (int)
     int capacity = 59;
-    //     5) Drag from fuselage (float)
     float drag = 32.2f;
-    // 3 things it can do:
-    //     1) Enclose passenger cabin and cargo
+
     void encloseCabin();
-    //     2) Support wings and tail
     void supportWingsAndTail();
-    //     3) Maintain internal pressure
     void maintainInternalPressure();
 };
 
+void Fuselage::encloseCabin()
+{
+    std::cout << "Cabin enclosed." << std::endl;
+}
+void Fuselage::supportWingsAndTail()
+{
+    std::cout << "Supporting wings and tail." << std::endl;
+}
+void Fuselage::maintainInternalPressure()
+{
+    if(capacity >= 50)
+    {
+    std::cout << "Maintaining internal pressure." << std::endl;
+    }
+    else
+    {
+    std::cout << "Cannot maintain internal pressure. Increase capacity." << std::endl;
+    }
+}
```

Comment:
Code inside of Curly braces needs to be indented for spaces relative to those Curly braces
- [ ] revise